libs/atqr/domain/src/lib/valueObjects/index.ts

```diff
@@ -1,5 +1,4 @@
-export * from './contact' 
+export * from './contact'
 export * from './email'
+export * from './emailAddress'
 export * from './invitee'
-export * from './enrollment'
```

Comment:
why did you take enrollment and officiation value objects imports out of here?

Reply:
anywhere they are needed they can be imported by "@atqr/domain"

Reply:
I think the biggest question is why aren't they in the Value Objects folder anymore. But that isn't part of your task anyway :)